### PR TITLE
add test reader

### PR DESCRIPTION
**Pull request by agoalofalife**
I add new tests

### CODE REVIEW

**Comment by duncan3dc** [comment]
Hi @agoalofalife, can you explain a little about your motivation for these tests? What's the benefit of them?

File: src/Util/Reader/Stdin.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ protected function getStdIn()
 
         try {
             $this->setStdIn();
-        } catch (\Error $e) {
+        } catch (\Exception $e) {
```
Comment (duncan3dc) [inline src/Util/Reader/Stdin.php]: Why have you removed the support for handling PHP 7 errors?

**Comment by agoalofalife** [comment]
Hi @duncan3dc , I can't remember, probably to increase the coverage test

**Comment by duncan3dc** [comment]
Thanks for coming back to me @agoalofalife 

Unfortunately at the moment I can't merge these without an explanation, once I'm more familiar with these classes I'll take another look at this, thanks

**Pull request closed without merging**